spec/plugin_mixins/rabbitmq_connection_spec.rb

```diff
@@ -26,6 +26,18 @@ def register
   }
   let(:hare_info) { instance.instance_variable_get(:@hare_info) }
 
+  describe "rabbitmq_settings" do
+    let(:rabbitmq_settings) { super.merge({"connection_timeout" => 123, "heartbeat" => 456}) }
```

Comment:
omg this is amazing!

For anyone watching this -
http://rspec.info/blog/2013/02/rspec-2-13-is-released/
"let and subject declarations can use super" w/ examples.